### ISSUE TITLE
Can't send Manual Request with ZAP vD-2021-01-20

### ISSUE THREAD

**Issue by davewichers** [zaproxy/zaproxy#6426]
I was trying to test to verify issue #6045 was fixed properly (and I think it has been). To do this I was trying to replay the request to the very first test case, which is:

```http
POST https://localhost:8443/benchmark/pathtraver-00/BenchmarkTest00001 HTTP/1.1
User-Agent: Mozilla/5.0 (Windows NT 6.3; WOW64; rv:39.0) Gecko/20100101 Firefox/39.0
Pragma: no-cache
Cache-Control: no-cache
Content-Length: 0
Accept: text/html
Cookie: BenchmarkTest00001=JohnDoe
Host: localhost:8443
```

And I noticed 2 problems:
1) In the Manual Request editor, with the above in the window, when I hit 'Request' nothing happens.

And I see nothing in the console explaining why.

2) So I also tried changing the HTTP Method just to make sure something worked in that dialog.

And after trying to change it to a few different HTTP methods, the dialog just died, with some kind of warning about 'is the request' valid or something.

Once, I also got this stream of errors in the console, so this might be a clue as to what is the problem is:

<details>
<summary>Log of requests that failed to parse. (Click the triangle/control to expand.)</summary>

```
1511262 [AWT-EventQueue-0] ERROR org.parosproxy.paros.network.HttpRequestHeader - Failed to parse:
GET https HTTP/1.1
User-Agent: Mozilla/5.0 (Windows NT 6.3; WOW64; rv:39.0) Gecko/20100101 Firefox/39.0
Pragma: no-cache
Cache-Control: no-cache
Content-Length: 0
Accept: text/html
Cookie: BenchmarkTest00001=JohnDoe
Host: localhost:8443

org.apache.commons.httpclient.URIException: invalid port number
	at org.apache.commons.httpclient.URI.parseAuthority(URI.java:2269) ~[zap-D-2021-01-20.jar:?]
	at org.apache.commons.httpclient.URI.parseUriReference(URI.java:1999) ~[zap-D-2021-01-20.jar:?]
	at org.apache.commons.httpclient.URI.<init>(URI.java:181) ~[zap-D-2021-01-20.jar:?]
	at org.parosproxy.paros.network.HttpRequestHeader.parse(HttpRequestHeader.java:474) ~[zap-D-2021-01-20.jar:D-2021-01-20]
	at org.parosproxy.paros.network.HttpRequestHeader.setMessage(HttpRequestHeader.java:290) [zap-D-2021-01-20.jar:D-2021-01-20]
	at org.parosproxy.paros.network.HttpRequestHeader.setMessage(HttpRequestHeader.java:313) [zap-D-2021-01-20.jar:D-2021-01-20]
	at org.parosproxy.paros.network.HttpMessage.mutateHttpMethod(HttpMessage.java:1047) [zap-D-2021-01-20.jar:D-2021-01-20]
	at org.zaproxy.zap.extension.httppanel.HttpPanelRequest$1.actionPerformed(HttpPanelRequest.java:116) [zap-D-2021-01-20.jar:D-2021-01-20]
	at javax.swing.JComboBox.fireActionEvent(JComboBox.java:1264) [?:?]
	at javax.swing.JComboBox.setSelectedItem(JComboBox.java:589) [?:?]
	at javax.swing.JComboBox.setSelectedIndex(JComboBox.java:624) [?:?]
	at javax.swing.plaf.basic.BasicComboPopup$Handler.mouseReleased(BasicComboPopup.java:907) [?:?]
	at java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:298) [?:?]
	at java.awt.Component.processMouseEvent(Component.java:6632) [?:?]
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3342) [?:?]
	at com.apple.laf.AquaComboBoxPopup$1.processMouseEvent(AquaComboBoxPopup.java:178) [?:?]
	at java.awt.Component.processEvent(Component.java:6397) [?:?]
	at java.awt.Container.processEvent(Container.java:2263) [?:?]
	at java.awt.Component.dispatchEventImpl(Component.java:5008) [?:?]
	at java.awt.Container.dispatchEventImpl(Container.java:2321) [?:?]
	at java.awt.Component.dispatchEvent(Component.java:4840) [?:?]
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4918) [?:?]
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4547) [?:?]
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4488) [?:?]
	at java.awt.Container.dispatchEventImpl(Container.java:2307) [?:?]
	at java.awt.Window.dispatchEventImpl(Window.java:2772) [?:?]
	at java.awt.Component.dispatchEvent(Component.java:4840) [?:?]
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:772) [?:?]
	at java.awt.EventQueue$4.run(EventQueue.java:721) [?:?]
	at java.awt.EventQueue$4.run(EventQueue.java:715) [?:?]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:?]
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85) [?:?]
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95) [?:?]
	at java.awt.EventQueue$5.run(EventQueue.java:745) [?:?]
	at java.awt.EventQueue$5.run(EventQueue.java:743) [?:?]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:?]
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85) [?:?]
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:742) [?:?]
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203) [?:?]
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124) [?:?]
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113) [?:?]
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109) [?:?]
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101) [?:?]
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:90) [?:?]
1511281 [AWT-EventQueue-0] ERROR org.parosproxy.paros.network.HttpMessage - invalid port number
org.parosproxy.paros.network.HttpMalformedHeaderException: invalid port number
	at org.parosproxy.paros.network.HttpRequestHeader.setMessage(HttpRequestHeader.java:303) ~[zap-D-2021-01-20.jar:D-2021-01-20]
	at org.parosproxy.paros.network.HttpRequestHeader.setMessage(HttpRequestHeader.java:313) ~[zap-D-2021-01-20.jar:D-2021-01-20]
	at org.parosproxy.paros.network.HttpMessage.mutateHttpMethod(HttpMessage.java:1047) [zap-D-2021-01-20.jar:D-2021-01-20]
	at org.zaproxy.zap.extension.httppanel.HttpPanelRequest$1.actionPerformed(HttpPanelRequest.java:116) [zap-D-2021-01-20.jar:D-2021-01-20]
	at javax.swing.JComboBox.fireActionEvent(JComboBox.java:1264) [?:?]
	at javax.swing.JComboBox.setSelectedItem(JComboBox.java:589) [?:?]
	at javax.swing.JComboBox.setSelectedIndex(JComboBox.java:624) [?:?]
	at javax.swing.plaf.basic.BasicComboPopup$Handler.mouseReleased(BasicComboPopup.java:907) [?:?]
	at java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:298) [?:?]
	at java.awt.Component.processMouseEvent(Component.java:6632) [?:?]
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3342) [?:?]
	at com.apple.laf.AquaComboBoxPopup$1.processMouseEvent(AquaComboBoxPopup.java:178) [?:?]
	at java.awt.Component.processEvent(Component.java:6397) [?:?]
	at java.awt.Container.processEvent(Container.java:2263) [?:?]
	at java.awt.Component.dispatchEventImpl(Component.java:5008) [?:?]
	at java.awt.Container.dispatchEventImpl(Container.java:2321) [?:?]
	at java.awt.Component.dispatchEvent(Component.java:4840) [?:?]
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4918) [?:?]
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4547) [?:?]
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4488) [?:?]
	at java.awt.Container.dispatchEventImpl(Container.java:2307) [?:?]
	at java.awt.Window.dispatchEventImpl(Window.java:2772) [?:?]
	at java.awt.Component.dispatchEvent(Component.java:4840) [?:?]
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:772) [?:?]
	at java.awt.EventQueue$4.run(EventQueue.java:721) [?:?]
	at java.awt.EventQueue$4.run(EventQueue.java:715) [?:?]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:?]
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85) [?:?]
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95) [?:?]
	at java.awt.EventQueue$5.run(EventQueue.java:745) [?:?]
	at java.awt.EventQueue$5.run(EventQueue.java:743) [?:?]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:?]
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85) [?:?]
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:742) [?:?]
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203) [?:?]
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124) [?:?]
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113) [?:?]
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109) [?:?]
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101) [?:?]
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:90) [?:?]
1513317 [AWT-EventQueue-0] ERROR org.parosproxy.paros.network.HttpRequestHeader - Failed to parse:
GET https HTTP/1.1
User-Agent: Mozilla/5.0 (Windows NT 6.3; WOW64; rv:39.0) Gecko/20100101 Firefox/39.0
Pragma: no-cache
Cache-Control: no-cache
Content-Length: 0
Accept: text/html
Cookie: BenchmarkTest00001=JohnDoe
Host: localhost:8443
org.apache.commons.httpclient.URIException: invalid port number
	at org.apache.commons.httpclient.URI.parseAuthority(URI.java:2269) ~[zap-D-2021-01-20.jar:?]
	at org.apache.commons.httpclient.URI.parseUriReference(URI.java:1999) ~[zap-D-2021-01-20.jar:?]
	at org.apache.commons.httpclient.URI.<init>(URI.java:181) ~[zap-D-2021-01-20.jar:?]
	at org.parosproxy.paros.network.HttpRequestHeader.parse(HttpRequestHeader.java:474) ~[zap-D-2021-01-20.jar:D-2021-01-20]
	at org.parosproxy.paros.network.HttpRequestHeader.setMessage(HttpRequestHeader.java:290) [zap-D-2021-01-20.jar:D-2021-01-20]
	at org.parosproxy.paros.network.HttpRequestHeader.setMessage(HttpRequestHeader.java:313) [zap-D-2021-01-20.jar:D-2021-01-20]
	at org.parosproxy.paros.network.HttpRequestHeader.<init>(HttpRequestHeader.java:185) [zap-D-2021-01-20.jar:D-2021-01-20]
	at org.parosproxy.paros.network.HttpMessage.setRequestHeader(HttpMessage.java:400) [zap-D-2021-01-20.jar:D-2021-01-20]
	at org.zaproxy.zap.extension.httppanel.view.impl.models.http.request.RequestStringHttpPanelViewModel.setData(RequestStringHttpPanelViewModel.java:56) [zap-D-2021-01-20.jar:D-2021-01-20]
	at org.zaproxy.zap.extension.httppanel.view.syntaxhighlight.HttpPanelSyntaxHighlightTextView.save(HttpPanelSyntaxHighlightTextView.java:185) [zap-D-2021-01-20.jar:D-2021-01-20]
	at org.zaproxy.zap.extension.httppanel.component.HttpPanelComponentViewsManager.save(HttpPanelComponentViewsManager.java:300) [zap-D-2021-01-20.jar:D-2021-01-20]
	at org.zaproxy.zap.extension.httppanel.component.all.request.RequestAllComponent.save(RequestAllComponent.java:162) [zap-D-2021-01-20.jar:D-2021-01-20]
	at org.zaproxy.zap.extension.httppanel.HttpPanel.saveData(HttpPanel.java:361) [zap-D-2021-01-20.jar:D-2021-01-20]
	at org.zaproxy.zap.extension.httppanel.HttpPanelRequest$1.actionPerformed(HttpPanelRequest.java:97) [zap-D-2021-01-20.jar:D-2021-01-20]
	at javax.swing.JComboBox.fireActionEvent(JComboBox.java:1264) [?:?]
	at javax.swing.JComboBox.setSelectedItem(JComboBox.java:589) [?:?]
	at javax.swing.JComboBox.setSelectedIndex(JComboBox.java:624) [?:?]
	at javax.swing.plaf.basic.BasicComboPopup$Handler.mouseReleased(BasicComboPopup.java:907) [?:?]
	at java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:298) [?:?]
	at java.awt.Component.processMouseEvent(Component.java:6632) [?:?]
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3342) [?:?]
	at com.apple.laf.AquaComboBoxPopup$1.processMouseEvent(AquaComboBoxPopup.java:178) [?:?]
	at java.awt.Component.processEvent(Component.java:6397) [?:?]
	at java.awt.Container.processEvent(Container.java:2263) [?:?]
	at java.awt.Component.dispatchEventImpl(Component.java:5008) [?:?]
	at java.awt.Container.dispatchEventImpl(Container.java:2321) [?:?]
	at java.awt.Component.dispatchEvent(Component.java:4840) [?:?]
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4918) [?:?]
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4547) [?:?]
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4488) [?:?]
	at java.awt.Container.dispatchEventImpl(Container.java:2307) [?:?]
	at java.awt.Window.dispatchEventImpl(Window.java:2772) [?:?]
	at java.awt.Component.dispatchEvent(Component.java:4840) [?:?]
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:772) [?:?]
	at java.awt.EventQueue$4.run(EventQueue.java:721) [?:?]
	at java.awt.EventQueue$4.run(EventQueue.java:715) [?:?]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:?]
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85) [?:?]
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95) [?:?]
	at java.awt.EventQueue$5.run(EventQueue.java:745) [?:?]
	at java.awt.EventQueue$5.run(EventQueue.java:743) [?:?]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:?]
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85) [?:?]
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:742) [?:?]
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203) [?:?]
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124) [?:?]
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113) [?:?]
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109) [?:?]
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101) [?:?]
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:90) [?:?]
1513318 [AWT-EventQueue-0] WARN  org.zaproxy.zap.extension.httppanel.view.impl.models.http.request.RequestStringHttpPanelViewModel - Could not Save Header: GET https HTTP/1.1
User-Agent: Mozilla/5.0 (Windows NT 6.3; WOW64; rv:39.0) Gecko/20100101 Firefox/39.0
Pragma: no-cache
Cache-Control: no-cache
Content-Length: 0
Accept: text/html
Cookie: BenchmarkTest00001=JohnDoe
Host: localhost:8443
org.parosproxy.paros.network.HttpMalformedHeaderException: invalid port number
	at org.parosproxy.paros.network.HttpRequestHeader.setMessage(HttpRequestHeader.java:303) ~[zap-D-2021-01-20.jar:D-2021-01-20]
	at org.parosproxy.paros.network.HttpRequestHeader.setMessage(HttpRequestHeader.java:313) ~[zap-D-2021-01-20.jar:D-2021-01-20]
	at org.parosproxy.paros.network.HttpRequestHeader.<init>(HttpRequestHeader.java:185) ~[zap-D-2021-01-20.jar:D-2021-01-20]
	at org.parosproxy.paros.network.HttpMessage.setRequestHeader(HttpMessage.java:400) ~[zap-D-2021-01-20.jar:D-2021-01-20]
	at org.zaproxy.zap.extension.httppanel.view.impl.models.http.request.RequestStringHttpPanelViewModel.setData(RequestStringHttpPanelViewModel.java:56) [zap-D-2021-01-20.jar:D-2021-01-20]
	at org.zaproxy.zap.extension.httppanel.view.syntaxhighlight.HttpPanelSyntaxHighlightTextView.save(HttpPanelSyntaxHighlightTextView.java:185) [zap-D-2021-01-20.jar:D-2021-01-20]
	at org.zaproxy.zap.extension.httppanel.component.HttpPanelComponentViewsManager.save(HttpPanelComponentViewsManager.java:300) [zap-D-2021-01-20.jar:D-2021-01-20]
	at org.zaproxy.zap.extension.httppanel.component.all.request.RequestAllComponent.save(RequestAllComponent.java:162) [zap-D-2021-01-20.jar:D-2021-01-20]
	at org.zaproxy.zap.extension.httppanel.HttpPanel.saveData(HttpPanel.java:361) [zap-D-2021-01-20.jar:D-2021-01-20]
	at org.zaproxy.zap.extension.httppanel.HttpPanelRequest$1.actionPerformed(HttpPanelRequest.java:97) [zap-D-2021-01-20.jar:D-2021-01-20]
	at javax.swing.JComboBox.fireActionEvent(JComboBox.java:1264) [?:?]
	at javax.swing.JComboBox.setSelectedItem(JComboBox.java:589) [?:?]
	at javax.swing.JComboBox.setSelectedIndex(JComboBox.java:624) [?:?]
	at javax.swing.plaf.basic.BasicComboPopup$Handler.mouseReleased(BasicComboPopup.java:907) [?:?]
	at java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:298) [?:?]
	at java.awt.Component.processMouseEvent(Component.java:6632) [?:?]
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3342) [?:?]
	at com.apple.laf.AquaComboBoxPopup$1.processMouseEvent(AquaComboBoxPopup.java:178) [?:?]
	at java.awt.Component.processEvent(Component.java:6397) [?:?]
	at java.awt.Container.processEvent(Container.java:2263) [?:?]
	at java.awt.Component.dispatchEventImpl(Component.java:5008) [?:?]
	at java.awt.Container.dispatchEventImpl(Container.java:2321) [?:?]
	at java.awt.Component.dispatchEvent(Component.java:4840) [?:?]
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4918) [?:?]
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4547) [?:?]
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4488) [?:?]
	at java.awt.Container.dispatchEventImpl(Container.java:2307) [?:?]
	at java.awt.Window.dispatchEventImpl(Window.java:2772) [?:?]
	at java.awt.Component.dispatchEvent(Component.java:4840) [?:?]
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:772) [?:?]
	at java.awt.EventQueue$4.run(EventQueue.java:721) [?:?]
	at java.awt.EventQueue$4.run(EventQueue.java:715) [?:?]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:?]
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85) [?:?]
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95) [?:?]
	at java.awt.EventQueue$5.run(EventQueue.java:745) [?:?]
	at java.awt.EventQueue$5.run(EventQueue.java:743) [?:?]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:?]
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85) [?:?]
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:742) [?:?]
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203) [?:?]
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124) [?:?]
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113) [?:?]
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109) [?:?]
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101) [?:?]
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:90) [?:?]
```

</details>

**Comment by kingthorin** [comment]
"Request" is the pane/tab title, click the "Send" button:
![image](https://user-images.githubusercontent.com/7570458/106189100-d6569b00-6175-11eb-9195-1207787a211a.png)

```http
GET https HTTP/1.1
User-Agent: Mozilla/5.0 (Windows NT 6.3; WOW64; rv:39.0) Gecko/20100101 Firefox/39.0
Pragma: no-cache
Cache-Control: no-cache
Content-Length: 0
Accept: text/html
Cookie: BenchmarkTest00001=JohnDoe
Host: localhost:8443
```

Isn't a valid request, and yes I get the exact same error as you.

**Comment by davewichers** [comment]
On the Mac, this dialog looks much different, like so. And apparently I again got bit with the bug #5831 that I reported before, which isn't fixed on Mac, even though that aspect of the bug is closed. On my Mac, by default, the divider between these two panes is directly in the middle of the screen, so the SEND button is NOT visible (still). And boy does the Request Label look like a BUTTON, esp. since there was no SEND button visible.

So, can you fix bug #5831 properly so on a Mac, the SEND but is ALWAYS visible by default. Maybe the problem is that Mac defaults to side-by-side displays of the request/response panes, and should instead default to top-bottom like your example?
And 2, it might be helpful to disable the Request/Response labels (on Mac) in some way so they don't look like the buttons. But if the SEND button is always visible that wouldn't be as big of a deal.

<img width="1223" alt="Screen Shot 2021-01-28 at 2 49 14 PM" src="https://user-images.githubusercontent.com/5004037/106190742-128afb00-6178-11eb-8584-7801eb660b80.png">

**Comment by thc202** [comment]
This is actually a duplicate of #5831. It's the same problem in all OSes.

Regarding point 2, we don't control how the buttons are rendered that depends on the look and feel. Might worth trying one of the other look and feels (you can dynamically change them with the button in the tool bar).
